### DISCUSSION TITLE
ipq60XX 支持 1G 内存的版本吗？

### DISCUSSION THREAD

**Discussion by tanst** [coolsnowwolf/lede#11844] [Q&A / 快速问答]
想问 3 个问题，谢谢：
1. zn m2 和 CMIOT AX18 都增加到了 1G 内存，直接选设备的话支持吗？
2. 还有怎么关闭无线 WIFI 呢？
3. 默认 .config 直接可以编译 ipq60xx 吗？

**Comment by ykxgt** [comment]
1. ipq6000内存和固件没关系，需要刷对应的cdt文件。
2. 据我知道，要彻底关闭wifi需要编译nowifi版本，具体怎么编译我不晓得。
3. 不知道你的问题在说什么。

**Comment by liyiranlab** [comment]
都1g ram了还在乎wifi的那点占用。
